src/generated/xproto.rs

```diff
@@ -7811,163 +9365,176 @@ pub struct ConfigureWindowAux {
         self.width = value.into();
         self
     }
-    /// Set the height field of this structure.
     pub fn height<I>(mut self, value: I) -> Self where I: Into<Option<u32>> {
         self.height = value.into();
         self
     }
-    /// Set the border_width field of this structure.
     pub fn border_width<I>(mut self, value: I) -> Self where I: Into<Option<u32>> {
         self.border_width = value.into();
         self
     }
-    /// Set the sibling field of this structure.
     pub fn sibling<I>(mut self, value: I) -> Self where I: Into<Option<Window>> {
         self.sibling = value.into();
         self
     }
-    /// Set the stack_mode field of this structure.
-    pub fn stack_mode<I>(mut self, value: I) -> Self where I: Into<Option<u32>> {
+    pub fn stack_mode<I>(mut self, value: I) -> Self where I: Into<Option<StackMode>> {
         self.stack_mode = value.into();
         self
     }
 }
-impl Serialize for ConfigureWindowAux {
-    type Bytes = Vec<u8>;
-    fn serialize(&self) -> Vec<u8> {
-        let mut result = Vec::new();
-        self.serialize_into(&mut result);
-        result
-    }
-    fn serialize_into(&self, bytes: &mut Vec<u8>) {
-        if let Some(ref value) = self.x {
-            value.serialize_into(bytes);
-        }
-        if let Some(ref value) = self.y {
-            value.serialize_into(bytes);
-        }
-        if let Some(ref value) = self.width {
-            value.serialize_into(bytes);
-        }
-        if let Some(ref value) = self.height {
-            value.serialize_into(bytes);
-        }
-        if let Some(ref value) = self.border_width {
-            value.serialize_into(bytes);
-        }
-        if let Some(ref value) = self.sibling {
-            value.serialize_into(bytes);
-        }
-        if let Some(ref value) = self.stack_mode {
-            value.serialize_into(bytes);
-        }
-    }
-}
+
 /// Configures window attributes.
 ///
+/// 
 /// Configures a window's size, position, border width and stacking order.
+///       
 ///
 /// # Fields
 ///
+/// * `window` - The window to configure.
+/// * `value_mask` - Bitmask of attributes to change.
 /// * `value_list` - New values, corresponding to the attributes in value_mask. The order has to
 /// correspond to the order of possible `value_mask` bits. See the example.
-/// * `value_mask` - Bitmask of attributes to change.
-/// * `window` - The window to configure.
 ///
 /// # Errors
 ///
-/// * `Match` - You specified a Sibling without also specifying StackMode or the window is not
-///actually a Sibling.
-/// * `Value` - TODO: reasons?
-/// * `Window` - The specified window does not exist. TODO: any other reason?
+/// * `Match` - 
+/// You specified a Sibling without also specifying StackMode or the window is not
+/// actually a Sibling.
+///       
+/// * `Window` - 
+/// The specified window does not exist. TODO: any other reason?
+///       
+/// * `Value` - 
+/// TODO: reasons?
+///       
```

Comment:
There are lots of small changes here in the docs. The new generator seems to include line breaks at the beginning and end of the individual XML items. This is harmless in most places, but here it looks really weird.

xcbgen seems to strip all the "doc stuff" it gets from the XML:
https://github.com/psychon/x11rb/blob/787b0955bbf73ecd542f0929a3d48102d63eb587/xcb-proto-1.14-1-g2b3559c/xcbgen/xtypes.py#L1380-L1393
From a quick look at Google, python's `strip()` seems to be Rust's `trim()`.

Reply:
I was doing it for part of the docs, but I missed the errors, fixed.